packages/@aws-cdk/aws-scheduler-targets-alpha/lib/sqs-send-message.ts

```diff
@@ -0,0 +1,76 @@
+import { ISchedule, IScheduleTarget, ScheduleTargetConfig } from '@aws-cdk/aws-scheduler-alpha';
+import { Names } from 'aws-cdk-lib';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+import * as sqs from 'aws-cdk-lib/aws-sqs';
+import { ScheduleTargetBase, ScheduleTargetBaseProps } from './target';
+import { sameEnvDimension } from './util';
+
+/**
+ * Properties for a SQS Queue Target
+ */
+export interface SqsSendMessageProps extends ScheduleTargetBaseProps {
+  /**
+   * The FIFO message group ID to use as the target.
+   *
+   * This must be specified when the target is a FIFO queue. If you specify
+   * a FIFO queue as a target, the queue must have content-based deduplication enabled.
+   *
+   * A length of `messageGroupId` must be between 1 and 128.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-scheduler-schedule-sqsparameters.html#cfn-scheduler-schedule-sqsparameters-messagegroupid
+   *
+   * @default - no message group ID
+   */
+  readonly messageGroupId?: string;
+}
+
+/**
+ * Use an Amazon SQS Queue as a target for AWS EventBridge Scheduler.
+ */
+export class SqsSendMessage extends ScheduleTargetBase implements IScheduleTarget {
+  constructor(
+    private readonly queue: sqs.IQueue,
+    private readonly props: SqsSendMessageProps,
+  ) {
+    super(props, queue.queueArn);
+
+    if (props.messageGroupId !== undefined) {
+      if (props.messageGroupId.length < 1 || props.messageGroupId.length > 128 ) {
+        throw new Error(`messageGroupId length must be between 1 and 128, got ${props.messageGroupId.length}`);
+      }
```

Comment:
I don't think tokens ever get this long but maybe:

```suggestion
      if (!Token.isUnresolved(props.messageGroupId) && (props.messageGroupId.length < 1 || props.messageGroupId.length > 128)) {
        throw new Error(`messageGroupId length must be between 1 and 128, got ${props.messageGroupId.length}`);
      }
```

Reply:
Yes, I'll do that just in case.